people/Turbo87.toml

```diff
@@ -3,3 +3,4 @@ github = "Turbo87"
 github-id = 141300
 email = "tobias.bieniek@gmail.com"
 discord-id = 313625269405745155
+zulip-id = 282338
```

Comment:
I don't seem to have write access here so I can't check the checkbox, but I can confirm that this ID is correct :)